Lib/idlelib/idle_test/test_squeezer.py

```diff
@@ -18,6 +19,15 @@
 
 SENTINEL_VALUE = sentinel.SENTINEL_VALUE
 
+def get_font_family():
```

Comment:
Perhaps a more descriptive name?

```suggestion
def get_fixed_width_font_family():
```

Reply:
I propose a shorter name :-) get_fixed_font_family

Reply:
The docstring gives the function's contract.  Any of the 'Tk...' font names predefined *on all systems*, such as 'TkDefaultFont' should work as well.  But 'TkFixedFont' is IDLE's default font and already used in the config code (writtin by Serhiy Storchake) I copied.